ml-agents/mlagents/trainers/trajectory.py

```diff
@@ -11,6 +13,17 @@
 from mlagents.trainers.torch.action_log_probs import LogProbsTuple
 
 
+class GroupmateStatus(NamedTuple):
```

Comment:
What could be alternative names for this class? I think Groupmate is not needed, this does not need to be for a teammate right? This is how we use it here, but it could be used as a general AgentStatus. So I would name it AgentStatus or something similar.

Reply:
This might be a good candidate to combine with the [DemonstrationExperience](https://github.com/Unity-Technologies/ml-agents/pull/4988/files#diff-66fb6b6106fe2d4c3010a78129be3eb9bb2e722df2638ceaf6eb109657437a86R13). There seems to be a common need for an AgentExperience that doesn't contain all of the policy-related information (e.g. log_probs). I can think of three options:

- Have a base AgentStatus with the minimal information, and inherit the larger AgentExperience from that. 
- Have different classes for each of the use-cases (which is what we have now with these two PRs)
- Have a single large AgentExperience with Optional fields

cc: @chriselion

Reply:
Renamed it to AgentStatus for now.